build-scripts/build-catch.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 
 # Install Catch
-if [[ -f ./catch/catch.hpp ]]; then
+if [[ -f ./catch/single_include/catch2/catch.hpp ]]; then
     echo "Catch has already been download and installed"
 else
     echo "Installing Catch"
     if [[ ! -d "./catch" ]]; then
-        wget https://github.com/catchorg/Catch2/releases/download/v2.13.6/catch.hpp -P catch
+        git clone https://github.com/catchorg/Catch2.git catch
```

Comment:
It's a minor optimization, but it is probably more efficient to just pull down what we need with a shallow clone:

```bash
git clone https://github.com/catchorg/Catch2.git catch --branch v2.13.6 --depth=1
```

Reply:
Updated in cfe329a